src/main/java/com/kodilla/ecommercee/domain/cart/CartDto.java

```diff
@@ -18,4 +18,7 @@
     private List<Product> products;
     private UserEntity user;
 
+    public void setUser(UserEntity user) {
+        this.user = user;
```

Comment:
skoro korzystasz z lomboka to zawsze możesz dodać @Setter zamiast pojedynczego settera.

Reply:
Teraz zauważyłem, że ja CartDto nigdzie nie używam, stworzyłem go na początku i tak został... usunę go